lua/wikis/ageofempires/Infobox/Map/Custom.lua

```diff
@@ -72,21 +79,20 @@ end
 ---@param args table
 ---@return table
 function CustomMap:addToLpdb(lpdbData, args)
-	lpdbData.extradata = {
+	lpdbData.extradata = Table.merge(lpdbData.extradata, {
 		creator = String.isNotEmpty(args.creator) and mw.ext.TeamLiquidIntegration.resolve_redirect(args.creator) or nil,
 		spawns = args.players,
 		maptype = self:_getType(args.type),
 		icon = args.icon,
-		game = Game.name{game = args.game}
-	}
+	})
 	return lpdbData
 end
 
 ---@param args table
 ---@return string[]
 function CustomMap:getWikiCategories(args)
 	return {
-		Game.name{game = args.game} .. ' Maps',
+		self:getGame(args) .. ' Maps',
```

Comment:
can error for unset game input!

Reply:
I'd prefer it to error even (for AoE), might add a decent error message to it

Reply:
a descriptive error message is preferable over a nil concat error message
so imho make it an assert
```suggestion
		assert(self:getGame(args), 'Missing or invalid game specified') .. ' Maps',
```

Reply:
Don't make an inline assert though, that is just unreadable